### DISCUSSION TITLE
Can't search result for pekko-connectors-slick in maven center?

### DISCUSSION THREAD

**Discussion by laglangyue** [apache/pekko-connectors#291] [General]
I can searh pekko-connectors-jms in maven center.
![image](https://github.com/apache/incubator-pekko-connectors/assets/35491928/c51548db-429e-4148-808b-a8c11691813b)

bug I can't serarch pekko-connectors-slick,I can get it by spell the url https://mvnrepository.com/artifact/org.apache.pekko/pekko-connectors-slick.

![image](https://github.com/apache/incubator-pekko-connectors/assets/35491928/419df2fd-4e09-4bb6-8e58-f5a00b48256c)

and  same  for pekko-connector-elasticsearch.
I once had thought pekko-connectors-slicl=1.0.1 that was not released.

**Comment by laglangyue** [comment]
IIt seems that some pekko connectors can be directly searched for, while others cannot. I think the search-result maybe cause some mistakes,some users would missing the released version such as me.

**Reply by pjfanning** [reply]
That tool has absolutely nothing to do with us. If you have a problem with that tool, talk to the maintainers of the tool.

**Reply by pjfanning** [reply]
https://mvnrepository.com/ is not maven central - it is a web site maintained by an individual.

**Reply by pjfanning** [reply]
https://search.maven.org/ is an official Maven web site.
